articles/search/cognitive-search-how-to-debug-skillset.md

```diff
@@ -153,15 +153,35 @@ If skills produce output but the search index is empty, check the field mappings
 
 Custom skills can be more challenging to debug because the code runs externally, so the debug session can't be used to debug them. This section describes how to locally debug your Custom Web API skill, debug session, Visual Studio Code and [ngrok](https://ngrok.com/docs). This technique works with custom skills that execute in [Azure Functions](../azure-functions/functions-overview.md) or any other Web Framework that runs locally (for example, [FastAPI](https://fastapi.tiangolo.com/)).
```

Comment:
line 154 - the first reference to ngrok is in the section intro, so I think Tunnelmole should be here also:  "and [ngrok](https://ngrok.com/docs) or [Tunnelmole](https://tunnelmole.com/)"